### ISSUE TITLE
Change Volume Mounts

### ISSUE THREAD

**Issue by jgomezve** [mongodb/mongodb-kubernetes-operator#1109]
**What did you do to encounter the bug?**
Is it possible to only create PVC for the data and not for the logs.

**Comment by tomislater** [comment]
It would be nice to have such option. Most of use store logs in ElasticSearch for example.

**Comment by tomislater** [comment]
It should not be closed.

**Comment by dan-mckean** [comment]
Hi, I'm Dan and I'm the Product Manager for MongoDB's support of Kubernetes.

I'm doing some work right now to try and identify how the Community Operator is being used. The Community Operator is something I inherited when I started at MongoDB, but it doesn't get as much attention from us as we'd like and we're trying to understand how it's used in order to establish its future. It will help us prioritize future issues and PRs raised by the community 🙂

Here's a super short survey (it's much easier for us to review all the feedback that way!): [https://docs.google.com/forms/d/e/1FAIpQLSfwrwyxBSlUyJ6AmC-eYlgW_3JEdfA48SB2i5--_WpiynMW2w/viewform?usp=sf_link](https://docs.google.com/forms/d/e/1FAIpQLSfwrwyxBSlUyJ6AmC-eYlgW_3JEdfA48SB2i5--_WpiynMW2w/viewform?usp=sf_link)

 If you'd rather email me instead: [dan.mckean@mongodb.com](mailto:dan.mckean@mongodb.com?subject=MongoDB%20Community%20Operator%20feedback)

Thank you in advance!
Dan
